smithy-aws-traits/src/main/java/software/amazon/smithy/aws/traits/apigateway/IntegrationTrait.java

```diff
@@ -290,7 +290,7 @@ public String getHttpMethod() {
      * @return Returns the request parameters.
      * @see <a href="https://docs.aws.amazon.com/apigateway/latest/developerguide/api-gateway-swagger-extensions-integration-requestParameters.html">Request parameters</a>
      */
-    public Map<String, String> getAllRequestParameters() {
+    public Map<String, String> getRequestParameters() {
```

Comment:
These method renames, in both classes, are breaking interface changes for the 0.9.x series that were intended as breaks for the 1.0.0 release. They should probably be removed from this for safety.

Reply:
Fixed in ec905353e3e2feedddd83a318278c6a94359e148